pub/errors/processor.php

```diff
@@ -501,13 +501,13 @@ public function saveReport(array $reportData): string
             $this->getReportDirNestingLevel($this->reportId),
             $this->reportId
         );
-        $reportDirName = dirname($this->_reportFile);
-        if (!file_exists($reportDirName)) {
-            @mkdir($reportDirName, 0777, true);
-        }
-        $this->_setReportData($reportData);
+//        $reportDirName = dirname($this->_reportFile);
+//        if (!file_exists($reportDirName)) {
+//            @mkdir($reportDirName, 0777, true);
+//        }
+//        $this->_setReportData($reportData);
 
-        @file_put_contents($this->_reportFile, $this->serializer->serialize($reportData). PHP_EOL);
```

Comment:
Reporting needed only in Development Env. 

For production better to use NewRelic APM/OpenTracing  sollution